common/components/controllers/MyProjectsController.jsx

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import type {Project} from '../stores/ProjectSearchStore.js';
+
+import ProjectAPIUtils from '../utils/ProjectAPIUtils.js';
+import MyProjectCard from '../componentsBySection/MyProjects/MyProjectCard.jsx';
```

Comment:
Is this file included in the changes?

Reply:
Ya -- all changes in this branch will be merged into master